Dispatcher/UI/Homepage/HomepageViewController.swift

```diff
@@ -8,26 +8,40 @@
 import UIKit
 import Alamofire
 
-class HomeViewController: UIViewController {
-    
-    @IBOutlet weak var searchButton: UIButton!
-    @IBOutlet weak var notificationsButton: UIButton!
+class HomepageViewController: UIViewController {
+
     @IBOutlet weak var articlesTableView: UITableView!
     @IBOutlet var homepageView: UIView!
     
-    let homepageViewModel = HomepageViewModel()
+    let homepageViewModel = HomepageViewModel.shared
     let activityIndicator = UIActivityIndicatorView()
+    var headerView: HeaderView?
     
     override func viewDidLoad() {
         super.viewDidLoad()
         self.navigationController?.isNavigationBarHidden = true
         
+        headerView = HeaderView(frame: CGRect(x: 0, y: 0, width: homepageView.frame.width, height: 95), headerType: .mainHeader)
```

Comment:
I'm curious why do you choose to add views programmatically in the code and not through the interface builder?\

Personally I prefer the opposite because then we know that when we look at the screen in the storyboard we can already know about all the elements that are in the page, and not some are in the storyboard and some are in the code. Consistency is important